.github/workflows/test.yml

```diff
@@ -69,8 +69,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: ["3.2", "3.1", "3.0", "2.7"]
-        sidekiq-version: ["~> 5.2", "~> 6.5", "~> 7"]
+        ruby-version: ["3.2", "3.1", "3.0"]
```

Comment:
I reduced the matrix size because ruby 2.7 has been EOL for a bit now and I think sidekiq v5 is also fairly old that we might want to drop it